include/rocksdb/options.h

```diff
@@ -1157,6 +1157,13 @@ struct DBOptions {
   //
   // Default: 1000000 (microseconds).
   uint64_t bgerror_resume_retry_interval = 1000000;
+
+  // A string identifying the location of the machine hosting the DB. This
+  // will be written as a property in every SST file written by the DB (or
+  // by offline writers such as SstFileWriter and RepairDB)
+  //
+  // Default: hostname
+  std::string db_host_location;
```

Comment:
Can you give a better comment as to what this would be used for?  Why would a user want to set this and why do we want this in the SST files?